src/rust/inetstack/protocols/layer3/arp/packet/message.rs

```diff
@@ -33,6 +35,7 @@ pub struct ArpMessage {
 impl ArpMessage {
     /// Creates an ARP message.
     pub fn new(pdu: ArpHeader) -> Result<Self, Fail> {
+        trace!("Creating ARP packet");
```

Comment:
Would printing some field of the header help in tracing?

Reply:
This is going away soon, so I didn't bother.